unblob/cli.py

```diff
@@ -6,14 +6,48 @@
 import click
 from structlog import get_logger
 
+from .dependencies import get_dependencies, is_installed, pretty_format_dependencies
+from .handlers import ALL_HANDLER_CLASS
 from .logging import configure_logger, noformat
 from .processing import DEFAULT_DEPTH, process_file
 from .state import exit_code_var
 
 logger = get_logger()
 
 
-@click.command()
+def show_external_dependencies(
+    ctx: click.Context, param: click.Option, value: bool
+) -> None:
+    if not value or ctx.resilient_parsing:
+        return
+
+    dependencies = get_dependencies(ALL_HANDLER_CLASS)
+    text = pretty_format_dependencies(dependencies)
+
+    exit_code = 0
+    for dependency in dependencies:
+        if not is_installed(dependency):
+            exit_code = 1
+            break
+
+    click.echo(text)
+    ctx.exit(code=exit_code)
+
+
+def get_help_text():
+    dependencies = get_dependencies(ALL_HANDLER_CLASS)
+    lines = [
+        "A tool for getting information out of any kind of binary blob.",
+        "",
+        "You also need these extractor commands to be able to extract the supported file types:",
+        ", ".join(dependencies),
+        "",
+        "NOTE: The extract extractor commands can be seen in the handlers, for version compatibility.",
```

Comment:
I added this note as we talked about this, but I created a separate issue to solve the versions: #134